### PR TITLE
added RVM Description to SMP Documentation

### LINKED ISSUE #102: [Docs]Add RVM Description to SMP Documentation
**Current State:**
Windows users are having issues using Jekyll and there's no description to help with that.

**Desired State:**
Provide a description of RVM to the `set up your development environment` section of the SMP Doc by adding this line.

` Note: Windows users can run Jekyll by following the [Windows Installation Guide](https://jekyllrb.com/docs/installation/windows/) and also install Ruby Version Manager [RVM](https://rvm.io) - a command-line tool which allows you to easily install, manage and work with multiple ruby environments on your local machine.Alternatively, if you're running Windows 10, updated to version 1903 Build 18362 or higher you can upgrade to Windows Subsystem for Linux [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10)`

The additional lines can be added right after *Jekyll Docs* .👍🏽

---
**Contributor Resources**
- [Service Mesh Performance documentation source](https://github.com/layer5io/service-mesh-performance/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/service-mesh-performance/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by oyinmi**
Signed-off-by: Oyinkansolami Atansuyi <tomisinatansuyi@gmail.com>

**Description**

This PR fixes #102 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Ruth-ikegah** [comment]
Nice work @oyinmi

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -44,6 +44,12 @@ Or you may configure your IDE, for example, Visual Studio Code to automatically
 
 <a href="https://user-images.githubusercontent.com/7570704/64490167-98906400-d25a-11e9-8b8a-5f465b854d49.png" ><img src="https://user-images.githubusercontent.com/7570704/64490167-98906400-d25a-11e9-8b8a-5f465b854d49.png" width="50%"><a>
 
+### Set up your development environment
+
+
+ Note: Windows users can run Jekyll by following the [Windows Installation Guide](https://jekyllrb.com/docs/installation/windows/) and also install Ruby Version Manager [RVM](https://rvm.io) - a command-line tool which allows you to easily install, manage and work with multiple ruby environments on your local machine. Alternatively, if you're running Windows 10, update to version 1903 Build 18362 or higher. You can also shift to Windows Subsystem for Linux [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
```
Comment (chandrashritii) [inline CONTRIBUTING.md:50]:
```suggestion
 Note: Windows users can run Jekyll by following the [Windows Installation Guide,](https://jekyllrb.com/docs/installation/windows/) for Jekyll. This includes installing the Ruby Version Manager [RVM](https://rvm.io) - a command-line tool that allows you to easily install, manage, and work with multiple ruby environments on your local machine. 
 Alternatively, if you are running Windows 10, you may install the Windows Subsystem for Linux:

-  [WSL1](https://docs.microsoft.com/en-us/windows/wsl/install-win10)  for Windows build version 1607 or higher.
-  [WSL2](https://docs.microsoft.com/en-us/windows/wsl/install-win10#update-to-wsl-2) for Windows build version 1903 or higher.

After successfully installation, proceed with the following [steps](https://jekyllrb.com/docs/installation/windows/#installation-via-bash-on-windows-10) to run Jekyll on WSL.
```

**Review by chandrashritii — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (chandrashritii) [reply CONTRIBUTING.md:50]: @oyinmi Good work! A few changes to make here.

Comment (oyinmi) [reply CONTRIBUTING.md:50]: @chandrashritii Thank you. Alright.

**Comment by oyinmi** [comment]
Change has been made. Thank you.

File: CONTRIBUTING.md (hunk 1 above)
Comment (oyinmi) [reply CONTRIBUTING.md]: @chandrashritii I've made the correction, thank you.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -44,6 +44,15 @@ Or you may configure your IDE, for example, Visual Studio Code to automatically
 
 <a href="https://user-images.githubusercontent.com/7570704/64490167-98906400-d25a-11e9-8b8a-5f465b854d49.png" ><img src="https://user-images.githubusercontent.com/7570704/64490167-98906400-d25a-11e9-8b8a-5f465b854d49.png" width="50%"><a>
 
+### Set up your development environment
+
+Note: Windows users can run Jekyll by following the [Windows Installation Guide,](https://jekyllrb.com/docs/installation/windows/) for Jekyll. This includes installing the Ruby Version Manager [RVM](https://rvm.io) - a command-line tool that allows you to easily install, manage, and work with multiple ruby environments on your local machine. 
+ Alternatively, if you are running Windows 10, you may install the Windows Subsystem for Linux:
```
Comment (chandrashritii) [inline CONTRIBUTING.md:50]:
```suggestion
 Alternatively, if you are running Windows 10, you may install the Windows Subsystem for Linux:
 
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -44,6 +44,15 @@ Or you may configure your IDE, for example, Visual Studio Code to automatically
...
+-  [WSL1](https://docs.microsoft.com/en-us/windows/wsl/install-win10)  for Windows build version 1607 or higher.
+-  [WSL2](https://docs.microsoft.com/en-us/windows/wsl/install-win10#update-to-wsl-2) for Windows build version 1903 or higher.
+After successfully installation, proceed with the following [steps](https://jekyllrb.com/docs/installation/windows/#installation-via-bash-on-windows-10) to run Jekyll on WSL.
```
Comment (chandrashritii) [inline CONTRIBUTING.md:53]:
```suggestion

After successfully installation, proceed with the following [steps](https://jekyllrb.com/docs/installation/windows/#installation-via-bash-on-windows-10) to run Jekyll on WSL.
```

Comment (oyinmi) [reply CONTRIBUTING.md:53]: Alright,  will fix it. Thank you

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -44,6 +44,15 @@ Or you may configure your IDE, for example, Visual Studio Code to automatically
 
 <a href="https://user-images.githubusercontent.com/7570704/64490167-98906400-d25a-11e9-8b8a-5f465b854d49.png" ><img src="https://user-images.githubusercontent.com/7570704/64490167-98906400-d25a-11e9-8b8a-5f465b854d49.png" width="50%"><a>
 
+### Set up your development environment
+
+Note: Windows users can run Jekyll by following the [Windows Installation Guide,](https://jekyllrb.com/docs/installation/windows/) for Jekyll. This includes installing the Ruby Version Manager [RVM](https://rvm.io) - a command-line tool that allows you to easily install, manage, and work with multiple ruby environments on your local machine.
```
Comment (JubayerJoy) [inline CONTRIBUTING.md:49]:
```suggestion
Note: Windows users can run Jekyll by following the [Windows Installation Guide](https://jekyllrb.com/docs/installation/windows/), for Jekyll. This includes installing the Ruby Version Manager [RVM](https://rvm.io) - a command-line tool that allows you to easily install, manage, and work with multiple ruby environments on your local machine. 
```

File: CONTRIBUTING.md (hunk 3 above)
Comment (JubayerJoy) [reply CONTRIBUTING.md:53]:
```suggestion
After successful installation, proceed with the following [steps](https://jekyllrb.com/docs/installation/windows/#installation-via-bash-on-windows-10) to run Jekyll on WSL.
```

**Review by JubayerJoy — COMMENTED** [review]
Very nice work @oyinmi! 🎈

File: CONTRIBUTING.md (hunk 4 above)
Comment (oyinmi) [reply CONTRIBUTING.md]: Thank you @JubayerJoy

**Review by PluckyPrecious — APPROVED** [review]
Welldone @oyinmi 🥳

**Review by kushthedude — COMMENTED** [review]
This change should go under `docs/` subdirectory of service-mesh-performance, where we have already specified how to contribute to docs please do the relevant changes!

**Comment by PluckyPrecious** [comment]
> This change should go under `docs/` subdirectory of service-mesh-performance, where we have already specified how to contribute to docs please do the relevant changes!

Absolutely, thanks @kushthedude for noticing. Welldone @oyinmi👍🏽 here’s the [docs](https://github.com/layer5io/service-mesh-performance/blob/master/docs/README.md)/README.md section the change is needed.

**Pull request merged**